src/js/containers/state/StateContainer.jsx

```diff
@@ -58,8 +58,10 @@ export class StateContainer extends React.Component {
             // Update the map center
             this.setStateCenter(this.props.params.stateId);
         }
-        if (!prevProps.params.fy && this.props.params.fy) {
-            // we just redirected the user to the new url which includes the fy selection
+        if (
+            (!prevProps.params.fy && this.props.params.fy) ||
```

Comment:
Couldn't we just keep the second part of the or statement? `prevProps.params.fy !== this.props.params.fy`